### ISSUE TITLE
Wheezy apt-get update extremely slow

### ISSUE THREAD

**Issue by arsu-leo** [debuerreotype/docker-debian-eol-artifacts#7]
Command:

> apt-get update

Takes several minutes to run, adding additional packages is just imposible.
Also happens with the jessie one.

> docker run --name test --rm -i -t debian/eol:wheezy
> \# apt-get update

**Comment by tianon** [comment]
I think you're probably hitting https://github.com/debuerreotype/docker-debian-artifacts/issues/53#issuecomment-436809949

**Comment by arsu-leo** [comment]
Seems that additional parameters doesnt fix the issue.
I have tried debian 11 and had no issue, also i could download ~20MB files from archive.debian.org
So it does't seem a connectivity issue outside the distribution:

> root@88f11d88b1c9:/# apt-get update
> Get:1 http://archive.debian.org wheezy Release.gpg [2373 B]
> 100% [Waiting for headers]
> Get:2 http://archive.debian.org wheezy/updates Release.gpg [1601 B]
> Get:3 http://archive.debian.org wheezy Release [191 kB]
> Get:4 http://archive.debian.org wheezy/updates Release [52.3 kB]
> Get:5 http://archive.debian.org wheezy/main amd64 Packages [7634 kB]
> 100% [Waiting for headers]
> Get:6 http://archive.debian.org wheezy/updates/main amd64 Packages [743 kB]
> Fetched 8624 kB in **10min 20s (13.9 kB/s)**

I had to activate the vsyscall=emulate on the wsl file as there was no way to make it work without it.

> PS C:\Users\MyUserName> docker --version
> Docker version 20.10.23, build 7155243   

Of course I'm on WSL 2:
> wsl -l -v
>  NAME                   STATE           VERSION
>  Debian                 Running         2
>  docker-desktop-data    Running         2
>  docker-desktop         Running         2

> docker version
> Client:
>  Cloud integration: v1.0.31
>  Version:           20.10.23
>  API version:       1.41
>  Go version:        go1.18.10
>  Git commit:        7155243
>  Built:             Thu Jan 19 17:43:10 2023
>  OS/Arch:           windows/amd64
>  Context:           default
>  Experimental:      true
> 
> Server: Docker Desktop 4.17.1 (101757)
>  Engine:
>   Version:          20.10.23
>   API version:      1.41 (minimum version 1.12)
>   Go version:       go1.18.10
>   Git commit:       6051f14
>   Built:            Thu Jan 19 17:32:04 2023
>   OS/Arch:          linux/amd64
>   Experimental:     false
>  containerd:
>   Version:          1.6.18
>   GitCommit:        2456e983eb9e37e47538f59ea18f2043c9a73640
>  runc:
>   Version:          1.1.4
>   GitCommit:        v1.1.4-0-g5fd4c4d
>  docker-init:
>   Version:          0.19.0
>   GitCommit:        de40ad0

**Comment by tianon** [comment]
I finally got a spare minute to test this, and I actually can't reproduce. :grimacing:

```console
$ docker pull debian/eol:wheezy
wheezy: Pulling from debian/eol
Digest: sha256:ea6f5f1e7186b95441ad601739332940f521db57a5b4dfcebcb564d814e4f918
Status: Image is up to date for debian/eol:wheezy
docker.io/debian/eol:wheezy
$ time docker run --rm debian/eol:wheezy apt-get update
Get:1 http://archive.debian.org wheezy Release.gpg [2373 B]
Get:2 http://archive.debian.org wheezy/updates Release.gpg [1601 B]
Get:3 http://archive.debian.org wheezy Release [191 kB]
Get:4 http://archive.debian.org wheezy/updates Release [52.3 kB]
Get:5 http://archive.debian.org wheezy/main amd64 Packages [7634 kB]
Get:6 http://archive.debian.org wheezy/updates/main amd64 Packages [743 kB]
Fetched 8624 kB in 2s (2967 kB/s)
Reading package lists...

real	0m4.058s
user	0m0.016s
sys	0m0.027s
```

**Comment by arsu-leo** [comment]
Docker desktop?

**Comment by tianon** [comment]
Nope, this was on a stock Debian Bullseye system (with `vsyscall=emulate` in my kernel command line).

**Comment by arsu-leo** [comment]
Is there anyways someone could try to reproduce this issue on windows through Docker Desktop?

> PS C:\Users\MyUserName> docker --version
> Docker version 20.10.23, build 7155243   

> wsl -l -v
>  NAME                   STATE           VERSION
>  Debian                 Running         2
>  docker-desktop-data    Running         2
>  docker-desktop         Running         2

> docker version
> Client:
>  Cloud integration: v1.0.31
>  Version:           20.10.23
>  API version:       1.41
>  Go version:        go1.18.10
>  Git commit:        7155243
>  Built:             Thu Jan 19 17:43:10 2023
>  OS/Arch:           windows/amd64
>  Context:           default
>  Experimental:      true
> 
> Server: Docker Desktop 4.17.1 (101757)
>  Engine:
>   Version:          20.10.23
>   API version:      1.41 (minimum version 1.12)
>   Go version:       go1.18.10
>   Git commit:       6051f14
>   Built:            Thu Jan 19 17:32:04 2023
>   OS/Arch:          linux/amd64
>   Experimental:     false
>  containerd:
>   Version:          1.6.18
>   GitCommit:        2456e983eb9e37e47538f59ea18f2043c9a73640
>  runc:
>   Version:          1.1.4
>   GitCommit:        v1.1.4-0-g5fd4c4d
>  docker-init:
>   Version:          0.19.0
>   GitCommit:        de40ad0
